<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRemoteDialog</class>
 <widget class="QDialog" name="ChangeRemoteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Remote</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>&amp;Url:</string>
     </property>
     <property name="buddy">
      <cstring>urlLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="urlLineEdit">
     <property name="placeholderText">
      <string>wss://example.org/qdsapp/</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="accessKeyLabel">
     <property name="text">
      <string>&amp;Access Key:</string>
     </property>
     <property name="buddy">
      <cstring>accessKeyLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="accessKeyLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Optional access secret</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="keepAliveTimeoutLabel">
     <property name="text">
      <string>Keep-&amp;Alive Timeout:</string>
     </property>
     <property name="buddy">
      <cstring>keepAliveTimeoutSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="keepAliveTimeoutSpinBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> min</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1440</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="keepDataLabel">
     <property name="text">
      <string>&amp;Keep data:</string>
     </property>
     <property name="buddy">
      <cstring>keepDataCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="keepDataCheckBox">
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTreeView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="extraHeadersLabel">
     <property name="text">
      <string>Extra &amp;Headers:</string>
     </property>
     <property name="buddy">
      <cstring>treeView</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QLineEdit" name="keyEdit">
       <property name="placeholderText">
        <string>Header Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valueEdit">
       <property name="placeholderText">
        <string>Header Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addHeaderButton">
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="removeHeaderButton">
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionA_dd_Header">
   <property name="icon">
    <iconset theme="list-add" resource="qtmvvmdatasyncwidgets_module.qrc">
     <normaloff>:/de/skycoder42/qtmvvm/widgets/icons/add.ico</normaloff>:/de/skycoder42/qtmvvm/widgets/icons/add.ico</iconset>
   </property>
   <property name="text">
    <string>A&amp;dd Header</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="action_Remove_Header">
   <property name="icon">
    <iconset theme="list-remove" resource="qtmvvmdatasyncwidgets_module.qrc">
     <normaloff>:/de/skycoder42/qtmvvm/widgets/icons/remove.ico</normaloff>:/de/skycoder42/qtmvvm/widgets/icons/remove.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Header</string>
   </property>
   <property name="toolTip">
    <string>Remove selected Header</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qtmvvmdatasyncwidgets_module.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangeRemoteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangeRemoteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
